src/modules/gifmaker/GifRecorderOverlay/Util/CaptureUtil.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GifRecorderOverlay.Util
+{
+    class CaptureUtil
+    {
+        CaptureUtil(
+            uint x,
+            uint y,
+            uint width,
+            uint height,
+            double framerate)
```

Comment:
If this is a Utility class then I think it should be `static`, otherwise rename possibly?